utils/log.go

```diff
@@ -86,6 +86,65 @@ func (l *LogLevel) UnmarshalText(text []byte) error {
 	return l.Set(string(text))
 }
 
+var ErrUnknownLogEncoding = fmt.Errorf(
+	"unknown log encoding (known: %s, %s)",
+	CONSOLE, JSON,
+)
+
+type LogEncoding int
+
+// The following are necessary for Cobra and Viper, respectively, to unmarshal log level
+// CLI/config parameters properly.
+var (
+	_ pflag.Value              = (*LogEncoding)(nil)
+	_ encoding.TextUnmarshaler = (*LogEncoding)(nil)
+)
+
+const (
+	CONSOLE LogEncoding = iota
```

Comment:
Can we rename it to `Console` please? according to standard go naming convention for const

Reply:
That isn't part of this PR, `CONSOLE` and some other constants in `utils/log.go` weren't created with this commit.  I suggest a separate issue/PR if you want to address that.